tools/run_tests/run_xds_tests.py

```diff
@@ -2595,7 +2595,12 @@ def __init__(self, compute, alpha_compute, project, project_num):
         client_env['GRPC_XDS_BOOTSTRAP'] = bootstrap_path
         client_env['GRPC_XDS_EXPERIMENTAL_CIRCUIT_BREAKING'] = 'true'
         client_env['GRPC_XDS_EXPERIMENTAL_ENABLE_TIMEOUT'] = 'true'
-        client_env['GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION'] = 'true'
+        # Temporarily turn off fault injection, because HTTPFault filter isn't
+        # handled correctly yet in CPP and Go. And setting would break all the
+        # other tests. Uncomment the following line when the support is
+        # complete.
+        #
+        # client_env['GRPC_XDS_EXPERIMENTAL_FAULT_INJECTION'] = 'true'
```

Comment:
Do we need an issue linked here in TODO? Just curious, which case isn't handled correctly in Cpp and Go?

Reply:
Go doesn't support Fault yet (PR is in progress).

Not sure about the exact failure in CPP, but all v3 tests are timing out.
Even v2 tests are failing (which also doesn't make sense, because v2 shouldn't be affected at all)

Reply:
That sounds really bad.

Reply:
You may want to manually run the interop tests and investigate a bit more.
Even simple tests like ping_pong are failing.